scripts/update_schemas_version.sh

```diff
@@ -1,4 +1,4 @@
-# https://gist.github.com/lukechilds/a83e1d7127b78fef38c2914c4ececc3c
+#!/bin/bash
```

Comment:
I think we should keep a link to the original author's gist.

Reply:
Done, thanks! 👍